src/React/DOM/Props.purs

```diff
@@ -99,13 +99,22 @@ coords = unsafeMkProps "coords"
 crossOrigin :: String -> Props
 crossOrigin = unsafeMkProps "crossOrigin"
 
+dataAttr :: String -> Props
```

Comment:
For these attributes you're asking about, I'm going by what's listed on https://facebook.github.io/react/docs/tags-and-attributes.html. Some of them I'm unfamiliar with as well.